conda/libmesh-vtk/conda_build_config.yaml

```
@@ -1,19 +1,16 @@
 moose_mpich:
-  - moose-mpich 4.0.2 build_16
+  - moose-mpich 4.0.2 build_17
 
 #### Darwin SDK SYSROOT
 CONDA_BUILD_SYSROOT:                                        # [osx]
-  - /opt/MacOSX10.15.sdk                                    # [not arm64 and osx]
-  - /opt/MacOSX11.3.sdk                                     # [arm64]
+  - /opt/MacOSX11.3.sdk                                     # [osx]
 
 macos_min_version:                                          # [osx]
-  - 10.15                                                   # [not arm64 and osx]
-  - 11.3                                                    # [arm64]
+  - 11.3                                                    # [osx]
 
 macos_machine:                                              # [osx]
   - x86_64-apple-darwin13.4.0                               # [not arm64 and osx]
```

```suggestion
  - x86_64-apple-darwin20.0.0                               # [not arm64 and osx]
```

If we're bumping to Big Sur as the minimum, this should also be adjusted.

---

That link suggests that it is used in the symbolic linking for the compiler binaries (that is, the name for the linked binary). Can you confirm whether this change is reflected in the `moose-mpich` compiler binaries?

---

But we're not actually building compilers in our Conda packages.

---

Exactly 😄 That's why I wanted you to tell me if it had *any* impact at all! I would doubt it based on that link.

---

well, when I grep for that variable, it is being consumed by none of our build.sh scripts. So I am guess we never were using it... Or needing to be setting it _again_ after it was being set correctly by the compiler activation scripts.

A full build in Civet will tell us :)